protos/module.proto

```diff
@@ -0,0 +1,10 @@
+syntax = "proto3";
+
+message ModuleInfo {
+  string name = 1;
+  string path = 2;
+  uint64 address_start = 3;
+  uint64 address_end = 4;
+  uint64 size = 5;
```

Comment:
What is the difference between `size` and `address_end - address_start (+1)`?

Reply:
The size is the file_size, it is (in most cases) different from address_start - end because of mapping gaps and .bss segment mapping rules. Maybe I should make it clear that this is file size.

Reply:
Yeah, then I would rename it to make it clearer.

Reply:
done.